Implementación de funcionalidad votación SI/NO

manmogvil opened the pull request:
Se ha implementado la votación dicotómica con distintas opciones y los test correspondientes

zorgary committed:
feat : Definir votación SI/NO #1

Realizado el formulario base para la creación de una votación SI/NO
y un pequeño template de prueba para comprobar su correcto
funcionamiento

manmogvil committed:
feat : Añadir tipos de preguntas dicotómicas #1

Añadido al formulario los siguientes tipos de respuestas
posibles : Si/No, Verdadero/Falso,
A favor/En contra y Bien/Mal

zorgary committed:
test : Crear tests para votaciones dicotómicas #1

Implementado test para comprobar la creación errónea de
votaciones dicotómicas

manmogvil committed:
test : Crear test votación #1

Creación de test positivos para la
votación dicotómica

manmogvil committed:
Merge branch 'feature/1-DefinirVotacionSiNo' of github.com:egc-veleta-g2/decide into feature/1-DefinirVotacionSiNo

zorgary reviewed (approved):
Todo correcto

Jesusjbs reviewed (changes requested):
Solucionar problemas con Codacy

manmogvil committed:
refactor : Corregir amenazas de codacy #1

Se han modificado las líneas de código que codacy ha recomendado como
vulnerabilidad
